firmware/octopi_firmware_v2/main_controller_teensy41/main_controller_teensy41.ino

```diff
@@ -124,7 +130,7 @@ static const int digitial_output_pins[num_digital_pins] = {2, 1, 6, 9, 10, 15};
 static const int camera_trigger_pins[] = {29, 30, 31, 32, 16, 28}; // trigger 1-6
 
 // motors
-const uint8_t pin_TMC4361_CS[4] = {41, 36, 35, 34};
+const uint8_t pin_TMC4361_CS[N_MOTOR] = {41, 36, 35, 34};
```

Comment:
@veerwang we should restore `const uint8_t pin_TMC4361_CS[4] = {41, 36, 35, 34};`

Reply:
In the future, suppose there are over 4 drivers slot in the device,  only change N_MOTOR to new value.